Bug -  PrefabStageUtility class is no longer Experimental

**Current Unity version used**
Unity 2021.3.0f1 LTS

**Describe the bug**
The following error is thrown when Unity 2021.3.0f1 LTS version+ is used. This is caused by PrefabStageUtility class no longer being Experimental.
```
Packages\HierarchyDecorator\HierarchyDecorator\Scripts\Editor\Hierarchy\Drawers\ToggleDrawer.cs(47,61): error CS0103: The name 'PrefabStageUtility' does not exist in the current context
```

--- a/HierarchyDecorator/Scripts/Editor/Hierarchy/Drawers/ToggleDrawer.cs
+++ b/HierarchyDecorator/Scripts/Editor/Hierarchy/Drawers/ToggleDrawer.cs
@@ -1,5 +1,4 @@
 ﻿using UnityEditor;
-using UnityEditor.Experimental.SceneManagement;
 using UnityEngine;
 
 namespace HierarchyDecorator
@@ -44,7 +43,12 @@ namespace HierarchyDecorator
 
         protected override bool DrawerIsEnabled(Settings _settings, GameObject instace)
         {
-           return _settings.globalData.showActiveToggles && PrefabStageUtility.GetCurrentPrefabStage() == null;
+#if UNITY_2021_2_OR_NEWER
+            var prefabStage = UnityEditor.SceneManagement.PrefabStageUtility.GetCurrentPrefabStage();
+#else
+            var prefabStage = UnityEditor.Experimental.SceneManagement.PrefabStageUtility.GetCurrentPrefabStage();
+#endif
+           return _settings.globalData.showActiveToggles && prefabStage == null;
         }
     }
 }

--- a/HierarchyDecorator/Scripts/Editor/Hierarchy/Drawers/ToggleDrawer.cs
+++ b/HierarchyDecorator/Scripts/Editor/Hierarchy/Drawers/ToggleDrawer.cs
@@ -48,7 +48,7 @@ namespace HierarchyDecorator
 #else
             var prefabStage = UnityEditor.Experimental.SceneManagement.PrefabStageUtility.GetCurrentPrefabStage();
 #endif
-           return _settings.globalData.showActiveToggles && prefabStage == null;
+            return _settings.globalData.showActiveToggles && prefabStage == null;
         }
     }
 }

